runtime/doc/filetype.txt

```diff
@@ -881,7 +881,7 @@ file: |ft_sql.txt|.
 
 TEX						*ft-tex-plugin* *g:tex_flavor*
 
-If the first line of a *.tex file has the form >
+If the first line of a "*.tex" file has the form >
```

Comment:
`*.tex`? (not sure how much you care about consistency here)

Reply:
Yeah this was intentional, I think quotes are what is intended for most of these cases, except when in a "table". Quotes also don't avoid parser errors if there is whitespace within the quotes.

Reply:
Sorry, markdown got interpreted, which made the comment... idiotic.

I meant everywhere else in this commit, you've used backticks around *.foo, while (only) here you use double quotes.

Reply:
fixed